Makefile

```
@@ -68,14 +68,16 @@ migrate: venv  ## Migrate the database
 	$(WITH_VENV) python app/manage.py migrate
 
 .PHONY: generate_models
-generate_models: generate_models_only
+generate_models: venv  ## Generate new app models.py file
+	$(WITH_VENV) python app/manage.py inspectdb --database milmove > new_models.py
+	mv new_models.py app/milmove_app/models.py
 	pre-commit run --all-files black || true
 	pre-commit run --all-files fix-encoding-pragma || true
 	@echo "Ignore errors from pre-commit, they are expected"
 
-.PHONY: generate_models_only
-generate_models_standalone: venv  ## Generate new app models.py file
-	$(WITH_VENV) python app/manage.py inspectdb --database milmove > new_models.py
+.PHONY: generate_models_standalone
+generate_models_standalone:
```

Help text is important too:

```suggestion
generate_models_standalone: ## Generate new app models.py file without virtualenv or pre-commit
```